about_processes.exs

```diff
@@ -0,0 +1,37 @@
+defmodule AboutProcesses do
+  use Koans
+
+  think "You are a process with a process ID" do
+    pid_name = inspect(self)
+
+    assert_? pid_name =~ ~r/#PID<\d+\.\d+\.\d+>/
```

Comment:
Maybe the first example should be a 

```
ref = spawn(fn -> IO.puts "I am an anonymous function" end)

assert_? is_pid(ref)
```

then you do one exercise that introduces `inspect(ref)` and `Process.info(ref)` and the you're blowing their mind by introducing `self`.